cpp-client/deephaven/dhclient/CMakeLists.txt

```diff
@@ -13,6 +13,38 @@ find_package(Protobuf CONFIG REQUIRED)
 find_package(gRPC CONFIG REQUIRED)
 find_package(Threads REQUIRED)
 
+set(PROTO_SRC_DIR
+  "${CMAKE_CURRENT_SOURCE_DIR}/../../../proto/proto-backplane-grpc/src/main/proto")
+set(PROTO_GEN_DIR
+  "${CMAKE_BINARY_DIR}/${PROJECT_NAME}/proto")
+set(PROTO_OUT_DIR "${PROTO_GEN_DIR}/deephaven/proto")
+
+file(GLOB PROTO_FILES "${PROTO_SRC_DIR}/deephaven/proto/*.proto")
```

Comment:
Just this one comment. I have, from the early days, preferred to list out every source file rather than using GLOB.

It's a stylistic preference which could go either way, but -- historically I have kind of preferred seeing the cmake file explicitly name every file it is working with.

(This is why I don't use GLOB for our .cc and .h files, even though we totally could)

Would you be willing to explicitly list the (eleven) proto files here?

Reply:
yep makes sense, we were doing that before so no problem